packages/dso-toolkit/src/components/card/card.args.ts

```diff
@@ -29,9 +25,7 @@ export const cardArgTypes: ArgTypes<CardArgs> = {
     },
   },
   href: {
-    control: {
-      type: "text",
-    },
+    ...noControl,
```

Comment:
Waarom is deze naar `...noControl` gegaan?

Reply:
JA, beetje in de war ivm verplichtheid van href ....